modules/local/qiime2_featuretable_group.nf

```diff
@@ -0,0 +1,41 @@
+process QIIME2_FEATURETABLE_GROUP {
+    tag "${category}"
+    label 'process_low'
+
+    conda (params.enable_conda ? { exit 1 "QIIME2 has no conda package" } : null)
+    container "quay.io/qiime2/core:2022.8"
+
+    input:
+    tuple path(table), path(metadata), val(category)
+
+    output:
+    path("${category}.qza")       , emit: qza
+    path "versions.yml" , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    """
+    export XDG_CONFIG_HOME="\${PWD}/HOME"
+
+    qiime feature-table filter-samples \
```

Comment:
```suggestion
    qiime feature-table filter-samples \\
```
You need double slashes if you want to actually put line breaks in the code here.

Reply:
If you don't have the double slashes, the code prints on one line with a long space in between. The single `\` just adds a line break in the multi-line groovy string, but it's saying this should still be all on one line. The double `\\` put's a literal `\` in the shell code, which the single slash doesn't